sdmetrics/reports/utils.py

```diff
@@ -154,7 +155,8 @@ def make_continuous_column_plot(real_column, synthetic_column, sdtype):
     """
     column_name = real_column.name if hasattr(real_column, 'name') else ''
     missing_data_real = round((real_column.isna().sum() / len(real_column)) * 100, 2)
-    missing_data_synthetic = round((synthetic_column.isna().sum() / len(synthetic_column)), 2)
+    missing_data_synthetic = round(
```

Comment:
Can we also add a check in the unit tests to make sure this is correct now?

Reply:
Yes I can